### PR TITLE
Pygtftk version 1.3.0

### PR DESCRIPTION
**Pull request by dputhier**
Hello,
This pull request is should provide Pygtftk version 1.3.0 for bioconda.
Best

----

Please read the [guidelines for Bioconda recipes](https://bioconda.github.io/contributor/guidelines.html) before opening a pull request (PR).

* If this PR adds or updates a recipe, use "Add" or "Update" appropriately as the first word in its title.
* New recipes not directly relevant to the biological sciences need to be submitted to the [conda-forge channel](https://conda-forge.org/docs/) instead of Bioconda.
* PRs require reviews prior to being merged. Once your PR is passing tests and ready to be merged, please issue the `@BiocondaBot please add label` command.
* Please post questions [on Gitter](https://gitter.im/bioconda/Lobby) or ping `@bioconda/core` in a comment.

<details>
  <summary>Please use the following BiocondaBot commands:</summary>

Everyone has access to the following BiocondaBot commands, which can be given in a comment:

<table>
  <tr>
    <td><code>@BiocondaBot please update</code></td>
    <td>Merge the master branch into a PR.</td>
  </tr>
  <tr>
    <td><code>@BiocondaBot please add label</code></td>
    <td>Add the <code>please review & merge</code> label.</td>
  </tr>
  <tr>
    <td><code>@BiocondaBot please fetch artifacts</code></td>
    <td>Post links to CI-built packages/containers. <br />You can use this to test packages locally.</td>
  </tr>
</table>

For members of the Bioconda project, the following command is also available:

<table>
  <tr>
    <td><code>@BiocondaBot please merge</code></td>
    <td>Upload built packages/containers and merge a PR. <br />Someone must approve a PR first! <br />This reduces CI build time by reusing built artifacts.</td>
  </tr>
</table>

Also, the bot watches for comments from non-members that include `@bioconda/<team>` and will automatically re-post them to notify the addressed `<team>`.

</details>

### CODE REVIEW

File: recipes/pygtftk/meta.yaml [hunk 1]
```diff
@@ -18,14 +18,14 @@ requirements:
...
-    - python
+    - python ==3.8
     - pip
     - numpy
     - cython
     - setuptools
     - zlib
   run:
-    - python
+    - python ==3.8
```
Comment (fanninpm) [inline recipes/pygtftk/meta.yaml:28]: These changes partially conflict with the contents of line 13.

File: recipes/pygtftk/meta.yaml [hunk 2]
```diff
@@ -1,11 +1,11 @@
...
+{% set version = "1.3.0" %}
 
 package:
   name: pygtftk
   version: '{{ version }}'
 
 source:
-  url: 'https://github.com/dputhier/pygtftk/archive/{{ version }}.tar.gz'
+  url: 'https://github.com/dputhier/pygtftk/archive/v{{ version }}.tar.gz'
   sha256: f3d58776a6ff3d79c43bf207a2157861ca2b5668f7f507dcca88994df6d77b4d
```
Comment (fanninpm) [inline recipes/pygtftk/meta.yaml:9]: Are you sure this hash was unchanged?

File: recipes/pygtftk/meta.yaml [hunk 3]
```diff
@@ -1,11 +1,11 @@
...
 
 build:
```
Comment (fanninpm) [inline recipes/pygtftk/meta.yaml:11]: I'm surprised the CI didn't complain about the build number not getting reset.

**Review by fanninpm — COMMENTED** [review]
@BiocondaBot please update

**Comment by fanninpm** [comment]
This recipe has been blacklisted. Try removing it from the blacklist.

https://github.com/bioconda/bioconda-recipes/blob/5cca17429f61426545dfde43bd8f903f1f8ed771/build-fail-blacklist#L122-L123

File: recipes/pygtftk/meta.yaml (hunk 2 above)
Comment (dputhier) [reply recipes/pygtftk/meta.yaml:9]: You are right. I forgot to update the hash...

File: recipes/pygtftk/meta.yaml [hunk 4]
```diff
@@ -1,16 +1,16 @@
...
   version: '{{ version }}'
 
 source:
-  url: 'https://github.com/dputhier/pygtftk/archive/{{ version }}.tar.gz'
-  sha256: f3d58776a6ff3d79c43bf207a2157861ca2b5668f7f507dcca88994df6d77b4d
+  url: 'https://github.com/dputhier/pygtftk/archive/v{{ version }}.tar.gz'
+  sha256: ca28ae5a394e1be422eeafbcd686e9cb616197c6758aa16049ac201d254ff61d
 
 build:
   number: 2
```
Comment (fanninpm) [inline recipes/pygtftk/meta.yaml:12]: This should satisfy the linter:
```suggestion
  number: 0
```

**Comment by fanninpm** [comment]
@BiocondaBot please add label

**Comment by dputhier** [comment]
@BiocondaBot please add label

**Review by dpryan79 — APPROVED** [review]

**Comment by dpryan79** [comment]
@bioconda-bot please merge

**Pull request merged**